anomaly_detection/cmd/api.py

```diff
@@ -25,7 +26,7 @@ def __init__(self):
         self._init_server()
 
     def _init_logging(self):
-        pass
+        log.setup(log.Config, "manila")
```

Comment:
Not "manila".
Please remove "manila" in all places.

Reply:
done